std/node/_fs/_fs_readlink_test.ts

```diff
@@ -1,18 +1,20 @@
 const { test } = Deno;
 import { readlink, readlinkSync } from "./_fs_readlink.ts";
 import { assertEquals, assert } from "../../testing/asserts.ts";
+import * as path from "../path.ts";
 
 const testDir = Deno.makeTempDirSync();
-const oldname = testDir + "/oldname";
-const newname = testDir + "/newname";
+const oldname = path.join(testDir, "oldname");
+const newname = path.join(testDir, "newname");
 
-if (Deno.build.os !== "windows") {
+if (Deno.build.os === "windows") {
+  Deno.symlinkSync(oldname, newname, { type: "file" });
+} else {
   Deno.symlinkSync(oldname, newname);
```

Comment:
can `{ type: "file" }` not be done in both windows and non-windows?

Reply:
This causes an error (on windows) because these paths do not actually exist and Deno.Symlink cant realize the target type.